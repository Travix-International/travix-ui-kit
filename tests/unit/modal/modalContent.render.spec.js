import { mount } from 'enzyme';
import React from 'react';
import ModalContent from '../../../components/modal/modalContent';

describe('Modal: render', () => {
  const requestAnimationFrame = global.window.requestAnimationFrame;

  beforeEach(() => {
    global.window.requestAnimationFrame = (cb) => { cb(); };
  });

  afterEach(() => {
    global.window.requestAnimationFrame = requestAnimationFrame;
  });

  it('Should redner modal content', () => {
    const component = mount(
      <ModalContent>
        Modal Content
      </ModalContent>
    );

    expect(component).toMatchSnapshot();
  });

  it('Should redner modal content with title', () => {
    const component = mount(
      <ModalContent title="Modal content title">
        Modal Content
      </ModalContent>
    );

    expect(component).toMatchSnapshot();
  });
});
