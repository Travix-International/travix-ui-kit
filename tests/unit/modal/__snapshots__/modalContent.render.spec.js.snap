// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal: render Should redner modal content 1`] = `
<ModalContent>
  <section
    className="ui-modal-content-block"
  >
    <div
      className="ui-modal-content-block__container"
    >
      Modal Content
    </div>
  </section>
</ModalContent>
`;

exports[`Modal: render Should redner modal content with title 1`] = `
<ModalContent
  title="Modal content title"
>
  <section
    className="ui-modal-content-block"
  >
    <div
      className="ui-modal-content-block__title"
    >
      Modal content title
    </div>
    <div
      className="ui-modal-content-block__container"
    >
      Modal Content
    </div>
  </section>
</ModalContent>
`;
